content/docs/install/access/index.md

```diff
@@ -58,18 +58,18 @@ This option exposes Keptn on each Kubernetes Node's IP at a static port. Therefo
 
 1. **Install Keptn:** Use the Keptn CLI to install Keptn on your cluster.
 If you want to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.
-  ```console
-  helm install keptn keptn/keptn -n keptn --version=$KeptnVersion --create-namespace --set=continuousDelivery.enabled=true,apiGatewayNginx.type=NodePort
-  ```
+    ```
+    helm install keptn keptn/keptn -n keptn --version=$KeptnVersion --create-namespace --set=continuousDelivery.enabled=true,apiGatewayNginx.type=NodePort
+    ```
 
-1. **Get Keptn endpoint:** Get the mapped port of the `api-gateway-nginx` using the command below.
+2. **Get Keptn endpoint:** Get the mapped port of the `api-gateway-nginx` using the command below.
```

Comment:
this is a markdown convention, please leave it in place. enumerations are always done with `1. `
```suggestion
1. **Get Keptn endpoint:** Get the mapped port of the `api-gateway-nginx` using the command below.
```